includes/cart/functions.php

```diff
@@ -233,8 +233,9 @@ function edd_add_to_cart( $download_id, $options = array() ) {
 		);
 	}
 
-	foreach ( $items as $item ) {
-		$to_add = apply_filters( 'edd_add_to_cart_item', $item );
+	foreach ( $items as &$item ) {
```

Comment:
Great idea to send the `$item` in as a reference so we're directly modifying it  with the filter. Can we be sure to put an `unset( $item );` call right after the foreach here then? 

They recommend this in the PHP docs:
`Warning: Reference of a $value and the last array element remain even after the foreach loop. It is recommended to destroy it by unset().`